### DISCUSSION TITLE
Data not showing in both the Preview and the published version of the app ("No rows found")

### DISCUSSION THREAD

**Discussion by egattulli** [Budibase/budibase#10163] [Help]
I'm creating a new app for my company with Budibase and SQL Server. I have already connected the DB and built a screen which displays the rows of the table "Users". To do that, I added a Data Provider and a Table. When I try to preview the app or open it, the table widget says "No rows found". This is weird. I can see the data only within the builder.

Does anyone know how to fix this? Thanks in advance!

**Comment by thanhpd56** [comment]
Me too.
